tests/suites/test_suite_psa_crypto.function

```diff
@@ -31,6 +31,29 @@
 #define ASSERT_OPERATION_IS_ACTIVE(   operation ) TEST_ASSERT( operation.id != 0 )
 #define ASSERT_OPERATION_IS_INACTIVE( operation ) TEST_ASSERT( operation.id == 0 )
 
+#if defined(PSA_WANT_ALG_JPAKE)
+void ecjpake_operation_setup( psa_pake_operation_t *operation,
+                              psa_pake_cipher_suite_t *cipher_suite,
+                              psa_pake_role_t role,
+                              mbedtls_svc_key_id_t key,
+                              size_t key_available )
+{
+    *operation = psa_pake_operation_init();
+
+    TEST_EQUAL( psa_pake_setup( operation, cipher_suite ),
+                PSA_SUCCESS );
```

Comment:
Non-blocker, here and elsewhere:
```suggestion
    PSA_ASSERT( psa_pake_setup( operation, cipher_suite ) );
```
This is normally equivalent, but `PSA_ASSERT` is clearer and is necessary for a feature we hardly ever use (`RECORD_PSA_STATUS_COVERAGE_LOG`).